tests/emails/beneficiary_activation_test.py

```diff
@@ -17,7 +17,7 @@
 class GetActivationEmailTest:
     def test_should_return_dict_when_environment_is_production(self):
         # Given
-        user = users_factories.UserFactory.build(email="fabien+test@example.net", firstName="Fabien")
+        user = users_factories.UserFactory.create(email="fabien+test@example.net", firstName="Fabien")
```

Comment:
(style)

```suggestion
        user = users_factories.UserFactory(email="fabien+test@example.net", firstName="Fabien")
```

idem en dessous.